Formula/erlang.rb

```diff
@@ -39,15 +43,14 @@ def install
     %w[LIBS FLAGS AFLAGS ZFLAGS].each { |k| ENV.delete("ERL_#{k}") }
 
     # Do this if building from a checkout to generate configure
-    system "./otp_build", "autoconf" if File.exist? "otp_build"
+    system "./otp_build", "autoconf" unless File.exist? "configure"
```

Comment:
Alternatively can check for `head`?

Based on https://github.com/erlang/otp/pull/4835, the  `configure` script will be checked into git
However, it may still be worth re-running `autoconf` on HEAD to get up-to-date `configure`...

Reply:
This is fine. No need to overthink what we're doing for HEAD. Priority is to fix the stable build.